fuel/transformers/__init__.py

```diff
@@ -118,8 +136,15 @@ def get_data(self, request=None):
 
 
 @add_metaclass(ABCMeta)
-class SingleMapping(Transformer):
-    """Applies a single mapping to multiple sources.
+class SourcewiseTransformer(Transformer):
```

Comment:
Similar comment about this class and `AgnosticTransformer` etc. I don't know, I'm interested in hearing what you think of that proposal first.